server/@types/express/index.d.ts

```diff
@@ -3,13 +3,16 @@ import { fieldErrors } from '../../middleware/validationMiddleware'
 
 export declare module 'express-session' {
   // Declare that the session will potentially contain these additional fields
+  import CreateContactJourney = journeys.CreateContactJourney
+
   interface SessionData {
     returnTo: string
     nowInMinutes: number
     prisonId: string
     prisonName: string
     search: string
     journey: object
+    createContactJourneys: Record<string, CreateContactJourney>
   }
```

Comment:
Does it need both `journey` and the list of journey types in `createContactJourneys`? Or does the latter replace the former? Might be my vague understanding of this..

Reply:
No it could be cleaned up, I left it there as wasn't sure who created it and what it was for.